Formula/go-kubectx.rb

```diff
@@ -0,0 +1,24 @@
+class GoKubectx < Formula
+  desc "Go-based kubectx alternative. 5x-10x faster"
```

Comment:
The benchmark results don't need to be in the description

Reply:
I'll update